src/main/java/com/FlightPub/model/Flight.java

```diff
@@ -105,43 +141,20 @@ public void setDeparture(String in){
         try {
             SimpleDateFormat originalFormat = new SimpleDateFormat("yyyy-MM-dd'T'hh:mm");
             Date date = originalFormat.parse(in);
-            this.departure = date;
+            this.departureTime = date;
         } catch (ParseException e) {
             System.out.println(e);
         }
     }
 
 
-    // Returns the Date in a readable String format
-    public String getArrivalTime(){
-        if(arrival == null)
+    public String getDepartureString(){
+        if(arrivalTime == null)
```

Comment:
should this be checking if the departureTime == null

Reply:
yup XD